__tests__/app.test.js

```diff
@@ -35,3 +36,37 @@ describe("/api", () => {
         });
     });
 });
+describe("/api/articles", () => {
+    test("GET: 200 sends an array of article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toHaveLength(13);
+          expect(articles).toBeInstanceOf(Array);
+          articles.forEach((article) => {
+            expect(article).toMatchObject({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+              comment_count: expect.any(String)
+            })
+        })
```

Comment:
Would be good to check that there is no key of body here too as toMatchObject is a flexible matcher and wouldn't pick up on that if it was included